lms/envs/aws.py

```diff
@@ -400,6 +399,13 @@
 
 AWS_STORAGE_BUCKET_NAME = AUTH_TOKENS.get('AWS_STORAGE_BUCKET_NAME', 'edxuploads')
 
+if AUTH_TOKENS.get('DEFAULT_FILE_STORAGE'):
```

Comment:
It's too bad we don't have an established way to generalize this common code -- just a comment, though, not a requirement for this PR